src/java.base/share/classes/java/time/Month.java

```diff
@@ -494,33 +476,21 @@ public int maxLength() {
      */
     public int firstDayOfYear(boolean leapYear) {
         int leap = leapYear ? 1 : 0;
-        switch (this) {
-            case JANUARY:
-                return 1;
-            case FEBRUARY:
-                return 32;
-            case MARCH:
-                return 60 + leap;
-            case APRIL:
-                return 91 + leap;
-            case MAY:
-                return 121 + leap;
-            case JUNE:
-                return 152 + leap;
-            case JULY:
-                return 182 + leap;
-            case AUGUST:
-                return 213 + leap;
-            case SEPTEMBER:
-                return 244 + leap;
-            case OCTOBER:
-                return 274 + leap;
-            case NOVEMBER:
-                return 305 + leap;
-            case DECEMBER:
-            default:
-                return 335 + leap;
-        }
+        return switch (this) {
+            case JANUARY   -> 1;
```

Comment:
The vertical alignment improves readability in these short-line cases. Removing the spaces before the arrows will make it a little harder to discern the difference between the cases.

Reply:
It is your codebase, not mine, so it is up to you. Aligning things by column is generally frowned on in most style guides because it handles refactoring poorly, resulting in lots of needless change (or people forgetting to realign things - I had to deal with a rogue aligned Javadoc signature today in a PR where exactly that had happened). I also don't see how you write a style guide rule for when these should be aligned and when they should not.

Anyway, this PR isn't really the right place for this discussion - I'm not blocking the PR on this basis (and I'm not an official Reviewer anyway).

Reply:
I have a slight preference for the aligned version in the cases where the lines are short and the number of spaces used for padding is not unreasonable. I find the code gains in readability in these cases. In the case where the line are long - I agree with Stephen that the alignment doesn't bring much readability - and sometime it may even be detrimental as it makes long lines longer. Stephen, would that be an acceptable compromise?

Reply:
Its certainly a reasonable position to take (ie. if thats what OpenJDK wants to do, its fine by me).
I'm more interested to see how you would write such a thing down in the coding standards that doesn't make the standard worthless.